src/systems/sensors/Sensors.cc

```diff
@@ -521,6 +628,52 @@ Sensors::~Sensors()
   this->dataPtr->Stop();
 }
 
+//TODO: why does math::Vector3d work but not math::Vector3i?
```

Comment:
looks like `math::Vector3i` is not one of the supported types: https://github.com/gazebosim/sdformat/blob/ba1bf8fedc0b6063d4312ad312e515e28f71cfb9/include/sdf/Param.hh#L529-L537